pkg/controller/queuejob/queuejob_controller_ex.go

```diff
@@ -460,9 +461,9 @@ func (qjm *XController) PreemptQueueJobs() {
 		klog.V(4).Infof("[PreemptQueueJobs] Adding preempted AppWrapper %s/%s to backoff queue.",
 			aw.Name, aw.Namespace)
 		go qjm.backoff(aw, "PreemptionTriggered", string(message))
-
```

Comment:
Nit: avoid formatting changes :)

Reply:
Mmm I don't care for that space, so it is gone